app/views/layouts/application.html.erb

```diff
@@ -16,6 +16,7 @@
     <link rel="apple-touch-icon" href="<%= image_path('apple-touch-icon.png') %>">
     <link rel="icon" href="<%= image_path('favicon.png') %>">
     <%= stylesheet_link_tag 'application' %>
+    <%= csrf_meta_tags %>
```

Comment:
Wow, how was this missing?

Reply:
I only noticed because I needed it for the local API :D
I don't think it's strictly necessary except for certain things like AJAX; your forms already had authenticity_tokens